packages/grid/src/Grid.tsx

```diff
@@ -1732,6 +1733,24 @@ class Grid extends PureComponent<GridProps, GridState> {
     }
   }
 
+  /**
+   * Handle a key up event from the keyboard. Pass the event to the registered keyboard handlers until one handles it.
+   * @param event Keyboard event
+   */
+  handleKeyUp(event: GridKeyboardEvent): void {
+    const keyHandlers = this.getKeyHandlers();
+    for (let i = 0; i < keyHandlers.length; i += 1) {
+      const keyHandler = keyHandlers[i];
+      const result = keyHandler.onUp(event, this);
+      if (result !== false) {
+        const options = result as EventHandlerResultOptions;
+        if (options?.stopPropagation ?? true) event.stopPropagation();
+        if (options?.preventDefault ?? true) event.preventDefault();
+        break;
+      }
+    }
```

Comment:
We could refactor this into another method `notifyKeyboardHandlers` similar to `notifyMouseHandlers` since both `handleKeyUp` and `handleKeyDown` have the same content other than the function name.

Also - looks like you didn't actually end up using this at all? Which is fine, may as well leave it in since it's already done, but ideally we'd have it as it's own PR.